environment/src/main/java/com/sequenceiq/environment/environment/service/EnvironmentService.java

```diff
@@ -207,9 +208,25 @@ public CreateAWSEnvironmentRequest getCreateAWSEnvironmentForCli(EnvironmentDto
         return environmentRepository.findByResourceCrnAndAccountIdAndArchivedIsFalse(resourceCrn, accountId);
     }
 
+    /**
+     * The CIDR could not be edited so the first step we clear the the existed CIDR every time.
```

Comment:
is it a hard requirement?

Reply:
what do you mean `hard`? Yes, if we edit a security group we always clear the cidr because we replace by a security group id